### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by MarkoKermi**
In this PR I manage to do the following requirements: 

- [ ] Implement the design from the sneak peek wireframes
- [ ] I create an `HTML` view for `user` and `post.`
- [ ] I create comment HTML file.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,13 @@
-<h2>This is one post from user!</h2>
\ No newline at end of file
+<%= render partial: 'user', locals: { user: @user}%>
+
+<article class="user_view">
+  <div class="bio_user">
+    <h4>Bio</h4>
+  </div>
```
Comment (tresorsawasawa) [inline app/views/users/show.html.erb]: - [ ] To match the provided [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) and as required for the project, kindly make sure you also render the `user's Bio` text as right now it is empty.

File: app/views/posts/_post.html.erb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <% end %>
+
+      <div>
+        <%# <% if truncated_message %>
+          <%# <%= post.text.truncate(20) %> 
+        <%# <% else %> 
+          <%# <%= post.text %> 
+        <%# <% end %> 
+      <%# </div> %> 
+
```
Comment (tresorsawasawa) [inline app/views/posts/_post.html.erb]: - [ ] For best practices and clean code, kindly make sure you remove all the commented and unused code lines from your code base

File: app/views/posts/_post.html.erb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<%= link_to user_post_path(post.author, post) do %>
+  <article class="post_view">
+    <div class="post__container">
+      <h3><%= post.title %></h3>
+
+      <% if local_assigns[:show_all_comments] %>
+        <div class="post__counters">
+          Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %>
+        </div>
+      <% end %>
```
Comment (tresorsawasawa) [inline app/views/posts/_post.html.erb:10]: - [ ] To match the provided **[wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)** and as required for the project, kindly make sure you also render the `post's text` text as right now nothing is displayed.

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -1 +1 @@
-<h2>This is one post from user!</h2>
\ No newline at end of file
+<%= render partial: 'post', locals: { post: @post, show_all_comments: true}%>
```
Comment (tresorsawasawa) [inline app/views/posts/show.html.erb]: - [ ] As required for the project, kindly ensure that the Post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png). Right now, when clicking on a post, your application raises the below error.
Screenshot from 2023-05-11 10-54-47

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @MarkoKermi 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @MarkoKermi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to make the necessary changes requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**